app/views/events/index.html.erb

```diff
@@ -1,5 +1,16 @@
-<% @events.each do |event| %>
-  <%= event.title %>
```

Comment:
peut être essaye de mettre un lien sur le titre vers la show ? qu'on voit si les liens entre nos parties marchent

Reply:
faut que j'en mettre un vers l'index aussi d'ailleurs

Reply:
Oui on fait le point ce matin après le live code tous ensemble sur la to do list de chaque teammate. Parmi les points à traiter, je te confirme l'insertion le link entre les differents pages pour un premier test 👌

Reply:
je te le merge pour le moment qu'on ait tous la première version de l'index ?

Reply:
Ok 👍